helm/values.schema.json

```diff
@@ -0,0 +1,1710 @@
+                    "type": "string",
+                    "title": "The host schema",
+                    "description": "Host name of the notary instance.",
+                    "examples": [
+                      "notary.docker.io"
+                    ],
+                    "pattern": "(([-a-zA-Z0-9@:%._\\+~#=]{1,256}\\.[a-zA-Z0-9()]{1,6})|(localhost))(\\:\\d{2,5})?"
+                  },
+                  "trust_roots": {
+                    "$id": "#/properties/validators/items/anyOf/0/properties/trust_roots",
+                    "type": [
+                      "array",
+                      "null"
+                    ],
+                    "title": "The trust_roots schema",
+                    "description": "The trust root of this validator. Currently, only public keys are supported as trust roots, but there may come other kinds of trust roots (e.g. mail addresses with OIDC flow).",
+                    "examples": [
+                      [
+                        {
+                          "name": "securesystemsengineering-official",
+                          "key": "-----BEGIN PUBLIC KEY-----\nMFkwEwYHKoZIzj0CAQYIKoZIzj0DAQcDQgAEsx28WV7BsQfnHF1kZmpdCTTLJaWe\nd0CA+JOi8H4REuBaWSZ5zPDe468WuOJ6f71E7WFg3CVEVYHuoZt2UYbN/Q==\n-----END PUBLIC KEY-----\n"
+                        }
+                      ]
+                    ],
+                    "items": {
+                      "$id": "#/properties/validators/items/anyOf/0/properties/trust_roots",
+                      "anyOf": [
+                        {
+                          "$id": "#/properties/validators/items/anyOf/0/properties/trust_roots/anyOf/0",
+                          "type": "object",
+                          "title": "The first anyOf schema",
+                          "properties": {
+                            "name": {
+                              "$id": "#/properties/validators/items/anyOf/1/properties/trust_roots/anyOf/0/properties/name",
+                              "type": "string",
+                              "title": "The trust root's name schema",
+                              "description": "Name of the trust root",
+                              "examples": [
+                                "securesystemsengineering-official"
+                              ]
+                            },
+                            "key": {
+                              "$id": "#/properties/validators/items/anyOf/1/properties/trust_roots/anyOf/0/properties/key",
+                              "type": "string",
+                              "title": "The trust root's key schema.",
+                              "pattern": "(?:-+BEGIN\\sPUBLIC\\sKEY[-]+)\n(?:(?:[A-Za-z0-9+\\/\\s])*={0,2})\n(?:-+END\\sPUBLIC\\sKEY[-]+)",
+                              "description": "Public ECDSA key used for verification of signatures on container images whose policy matches this trust root.",
```

Comment:
For you to ponder: Since the trust roots only says public key and the pattern doesn't limit it, is this the point to limit to ECDSA or would this be better generalized and docs the place to state ECDSA-only?